tests/runtests.sh

```diff
@@ -142,7 +142,7 @@ function start_selenium() {
 
     printf "\n\nStarting selenium...\n"
     if [ "$VNCPORT" != "" ]; then
-        docker run --rm=true -p 127.0.0.1:$VNCPORT:5900 \
+        docker run --rm=true -p $VNCPORT:5900 \
```

Comment:
I would rather localhost be the default. Is there a particular reason you did it the other way around?